src/__swaps__/screens/Swap/hooks/useSearchCurrencyLists.ts

```diff
@@ -320,6 +375,14 @@ export function useSearchCurrencyLists() {
     });
   }, [getRecentSwapsByChain, state.toChainId, query, memoizedData.keys, memoizedData.queryIsAddress]);
 
+  const popularAssetsForChain = useMemo(() => {
+    if (!popularAssets) return undefined;
+    if (!query) return popularAssets;
```

Comment:
we probably don't need this clause

Reply:
i included this bc when matching the list against a search query, the sort order can change, even if the search query is empty

Reply:
so the purpose is to preserve the sort order in that case

Reply:
I experienced this with the recents section. Ended up solving it by defining a sorter in the filterList